webcompat/templates/layout.html

```diff
@@ -13,7 +13,9 @@
   {% block extracss %}{% endblock %}
 
 </head>
-<body id="body-webcompat" data-username="{{ session.username }}">
+<body id="body-webcompat"
+  class="{%- if request.url_rule.endpoint == 'create_issue' and ab_active('exp') == 'form-v2' %}no-top-padding{% endif %}"
```

Comment:
Keep in mind that this doesn't only change the new form. It adds a class to every page pretty much. I would prefer if this wasn't fixed here but be contained in the codebase of the experiment.

Reply:
I filed https://github.com/webcompat/webcompat.com/issues/2955 to address this.